src/resqml2_0_1/RockFluidOrganizationInterpretation.h

```diff
@@ -0,0 +1,93 @@
+/*-----------------------------------------------------------------------
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"; you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-----------------------------------------------------------------------*/
+#pragma once
+
+#include "resqml2/AbstractGridRepresentation.h"
+#include "resqml2_0_1/AbstractOrganizationInterpretation.h"
+#include "resqml2_0_1/EarthModelInterpretation.h"
+#include "resqml2_0_1/OrganizationFeature.h"
+
+namespace RESQML2_0_1_NS
+{
+	/**
+	* This class is a container for other organizations that are consistent to each others.
+	*/
+	class DLL_IMPORT_OR_EXPORT RockFluidOrganizationInterpretation : public AbstractOrganizationInterpretation
+	{
+	public:
+		/**
+		* Only to be used in partial transfer context
+		*/
+		RockFluidOrganizationInterpretation(gsoap_resqml2_0_1::eml20__DataObjectReference* partialObject) : AbstractOrganizationInterpretation(partialObject) {}
+
+		/**
+		* Creates an instance of this class in a gsoap context.
+		* @param orgFeat			The feature the instance interprets. It must be a stratigraphic organization feature.
+		* @param guid				The guid to set to the interpretation. If empty then a new guid will be generated.
+		* @param title				A title for the instance to create.
+		* @param rank				The rank index of this interpretation within a stratigraphic column
+		* @param orderingCriteria	How the included horizons are ordered.
+		*/
+		RockFluidOrganizationInterpretation(class OrganizationFeature * orgFeat, const std::string & guid, const std::string & title, ULONG64 index);
```

Comment:
The index should not be part of the parameters. It must be arbitrary equal to the location of the unit in the organization.
By the way there is a known bug in 2.0.1 about adding fluid units in fuild organization : the workaround is not easy at all to implement!!!!! http://docs.energistics.org/#RESQML/RESQML_TOPICS/RESQML-500-106-0-R-sv2010.html

Reply:
Maybe we first make the simple case to work :)

Reply:
I agree but please : 

- remove the index from the method signature because it will NEVER be useful
- set it arbitrarily to 0 for now on the implementation side.

Reply:
still unresolved

Reply:
Actually it was resolved in commit 57b953a